docs/admin/release_notes/version_2.1.md

```diff
@@ -0,0 +1,20 @@
+
+## [v2.1.0 (2024-04-12)](https://github.com/nautobot/nautobot-app-floor-plan/releases/tag/v2.1.0)
+
+### Added
+
+- [#79](https://github.com/nautobot/nautobot-app-floor-plan/issues/79) - Added an option to change the grid column/row labels to use letters rather than numbers.
+- [#82](https://github.com/nautobot/nautobot-app-floor-plan/issues/82) - Added "floor plan tile" children count and hyperlink to the "floor plan" detail view for better navigation.
+
+### Changed
+
+- [#67](https://github.com/nautobot/nautobot-app-floor-plan/issues/67) - Replaced pydocstyle with ruff.
```

Comment:
nit: I'd consider this `housekeeping` as well, as it's not a user-facing change.